src/ReportManager/ReportDetailView/index.js

```diff
@@ -441,8 +441,7 @@ const ReportDetailView = ({
 
     setReportForm({ ...reportForm, event_details: { ...reportForm.event_details, ...formData } });
 
-    reportTracker.track('Change Report Form Data');
```

Comment:
Is removing this line intended?

Reply:
Yeah, it fires on every keystroke in the form and is spamming our analytics in a not-helpful way.